src/CarbonAware.WebApi/test/integrationTests/CarbonAwareControllerTests.cs

```diff
@@ -20,6 +20,8 @@ public class CarbonAwareControllerTests : IntegrationTestingBase
     private string bestLocationsURI = "/emissions/bylocations/best";
     private string currentForecastURI = "/emissions/forecasts/current";
     private string batchForecastURI = "/emissions/forecasts/batch";
+    private string actualURI = "/emissions/average-carbon-intensity";
```

Comment:
🐑 don't know how I feel about really calling average carbon intensity actual but don't feel strongly enough to change/have opinions on a new but also short name 🤷🏽‍♀️

Reply:
spitballing here, maybe
```suggestion
    private string actualHistoricalURI = "/emissions/average-carbon-intensity";
```